relay-server/src/utils/rate_limits.rs

```diff
@@ -125,6 +125,9 @@ pub struct EnvelopeSummary {
     /// The number of all session updates.
     pub session_quantity: usize,
 
+    /// The number of profiles.
+    pub profile_quantity: usize,
```

Comment:
Would it be possible we just introduce the data category, but not the counting? It's very likely that we will want to rate limit profiles by count; until we've pinned this down it would be good if there's no enforcement logic in Relay yet. 

The reason for this is that external downstream Relays might apply wrong rate limits if we were to change this.

Reply:
Discussed offline, we'll do this in one step.